CIP-0164/README.md

```diff
@@ -0,0 +1,2774 @@
+that workloads of 20,000e9 Plutus execution steps per EB may be feasible: this
+is 1000 times the current Cardano mainnet limit of 20e9 steps for Praos blocks.
+The subsequent plot shows the 4 vCPUs becoming progressively more saturated with
+heavier Plutus execution. Although these results suggest that Leios's
+_block-level_ Plutus budget can safely be 5000 billion steps or more, it is
+important to remember that this is for conditions where honest nodes faithfully
+and promptly diffuse the transactions requiring the relatively expensive phase 2
+(Plutus) validation: adversarial nodes could attempt to delay diffusion of
+transactions in order to overwhelm honest nodes with the sudden arrival of many
+heavy Plutus transactions and little time to validate them all before voting
+upon the newly seen EB. Experiments with prototype Leios nodes will be necessary
+to more precisely quantify how much the Plutus budget could safely be increased.
+
+<div align="center">
+<a name="figure-17" id="figure-17"></a>
+
+![Fate of Plutus-heavy transactions in Leios](images/plutus-temporal-efficiency-bar.svg)
+
+<em>Figure 17: Fate of Plutus-heavy transactions in Leios</em>
+
+</div>
+
+<div align="center">
+<a name="figure-18" id="figure-18"></a>
+
+|                                                                                             |                                                                                             |
+| ------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| ![Mean CPU usage in Plutus-heavy workloads for Leios](images/plutus-cpu-mean-histogram.svg) | ![Peak CPU usage in Plutus-heavy workloads for Leios](images/plutus-cpu-peak-histogram.svg) |
+
+<em>Figure 18: CPU usage in Plutus-heavy workloads for Leios</em>
+
+</div>
+
+In summary, Leios will require a modest increase of the [recommended hardware
+requirements][spohw]: a four-core machine will be required, but a network
+upgrade will not be needed, as 10 Mb/s is well below the bandwidth of standard
+network connections. At throughput much higher than 200 kB/s, network egress can
+become a significant cost for nodes hosted on some cloud-computing providers.
+The Leios simulations do not model memory or disk. With the advent of
+[UTxO-HD][utxohd], 16 GB of memory will remain be sufficient for Leios if the
+`OnDisk` option is used for the UTxO set. Disk requirements depend upon the
+growth of the ledger, but a sustained 0.150 MB/s throughput amounts to ledger
+size increasing by 4.7 TB each year: see the section below on Operating Costs
+for further discussion.
+
+### Feasible Protocol Parameters
+
+**Parameter Relationships and Network Assumptions**
+
+The key relation in the proposed protocol is between the voting threshold
+($\tau = 75\%$) and propagation delay of EBs ($\Delta_\text{EB}$). The high
```

Comment:
The % sign from tau does not render in my Debian desktop (Test here, `$100\%$`: $100\%$). Do you see it on your end?